src/webots/gui/WbAboutBox.cpp

```diff
@@ -24,38 +26,49 @@ WbAboutBox::WbAboutBox(QWidget *parent) : QDialog(parent) {
   setModal(true);
   setWindowTitle(tr("About Webots"));
   QPixmap webotsLogo("images:webots.png");
-  QGridLayout *layout = new QGridLayout();
-  layout->setColumnMinimumWidth(0, 150);
-  layout->setColumnMinimumWidth(1, 250);
-  layout->setRowMinimumHeight(1, 30);
+  QVBoxLayout *layout = new QVBoxLayout();
+  QGridLayout *firstRowLayout = new QGridLayout();
+  firstRowLayout->setColumnMinimumWidth(0, 150);
+  firstRowLayout->setColumnMinimumWidth(1, 250);
 
   QLabel *image = new QLabel();
   image->setPixmap(webotsLogo);
-  layout->addWidget(image, 0, 0, 1, 1, Qt::AlignCenter);
+  firstRowLayout->addWidget(image, 0, 0, 1, 1, Qt::AlignCenter);
 
-  int year = QDate::currentDate().year();
-  QLabel *licenseInfo = new QLabel();
-  connect(licenseInfo, &QLabel::linkActivated, &WbDesktopServices::openUrl);
-
-  licenseInfo->setWordWrap(true);
-  licenseInfo->setText(QString("<br> © Cyberbotics 1998-%1. Licensed under the <a style='color: #5DADE2;' "
-                               "href='https://www.apache.org/licenses/LICENSE-2.0'>Apache License, Version 2.0</a>.")
-                         .arg(year));
-  layout->addWidget(licenseInfo, 0, 1, 1, 1, Qt::AlignCenter);
+  QLabel *versionInfo = new QLabel();
+  connect(versionInfo, &QLabel::linkActivated, &WbDesktopServices::openUrl);
+  QDateTime releaseTimestamp;
+  releaseTimestamp.setTime_t(WbApplicationInfo::releaseDate());
+  versionInfo->setText(QString("<p style='font-size: large;'><b>%1</b></p>"
```

Comment:
It looks better to me if Webots is printed before the version number...
```suggestion
  versionInfo->setText(QString("<p style='font-size: large;'><b>Webots %1</b></p>"
```